### PR TITLE
Add 10th Birthday Blog article

### PR DESCRIPTION
**Pull request by chris-short**
_reviewer's note_: Add a blog article about Kubernetes' 10th birthday.

---

The Docs to Markdown extension generated some errors so I've cleaned those up and have a solid markdown file with no images at all

### CODE REVIEW

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:65]:
```suggestion
## Kubernetes today
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:73]:
```suggestion
## The future of Kubernetes
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:55]:
```suggestion
Kubernetes offers more extension points than we can count. Originally designed to work with Docker and only Docker, now you can plug in any container runtime that adheres to the CRI standard. There are other similar interfaces: CSI for storage and CNI for networking. And that's far from all you can do. In the last decade, whole new patterns have emerged, such as using [Custom Resource Definitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)  (CRDs) to support third-party controllers - now a huge part of the Kubernetes ecosystem.
```

**Review by sftim — DISMISSED** [review]
Mostly mechanical fixes and style nits.

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:53]: (nit)
```suggestion
**PS:** Curious to see how far the project has come for yourself? Check out this [tutorial for spinning up a Kubernetes 1.0 cluster](https://github.com/spurin/kubernetes-v1.0-lab) created by community members Carlos Santana, Amim Moises Salum Knabben, and James Spurin.
```

**Comment by chris-short** [comment]
@sftim I put the hold in place as a safety mechanism. Once we're all happy with what's here the hold can disappear. I suspect we'll unhold when we approve

**Comment by sftim** [comment]
/retitle [WIP] Add 10th Birthday Blog article

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (chris-short) [reply content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md]: Done!

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:45]: (nit)
```suggestion
* September 2019 — [CustomResourceDefinitions go GA](/blog/2019/09/18/kubernetes-1-16-release-announcement/) in Kubernetes 1.16.
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:40]: (nit)
```suggestion
  * This release also introduced StatefulSets and PodDisruptionBudgets in Beta.
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:54]:
```suggestion

---

```

(renders to `<hr/>`)

It'll make the tutorial stand out IMO.

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:55]: (nit)
```suggestion
Kubernetes offers more extension points than we can count. Originally designed to work with Docker and only Docker, now you can plug in any container runtime that adheres to the CRI standard. There are other similar interfaces: CSI for storage and CNI for networking. And that's far from all you can do. In the last decade, whole new patterns have emerged, such as using [CustomResourceDefinitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)  (CRDs) to support third-party controllers - now a huge part of the Kubernetes ecosystem.
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:42]:
```suggestion
* June 2017 — In [Kubernetes 1.7](/blog/2017/06/kubernetes-1-7-security-hardening-stateful-application-extensibility-updates/), ThirdPartyResources or "TPRs" are replaced with CustomResourceDefinitions (CRDs).
```

Optionally, ThirdPartyResource could link to https://github.com/kubernetes/design-proposals-archive/blob/main/api-machinery/extending-api.md

**Review by sftim — COMMENTED** [review]
More nit-level feedback. Doesn't block publication.

**Comment by sftim** [comment]
Still WIP? Does this need more work?

You can squash this down to 1 commit @chris-short but I won't in any way insist on it.

**Comment by chris-short** [comment]
@sftim @mrbobbytables I tried squashing it, but, of course, I did something wrong, and now it's gone. I then noticed Tim's additional change requests and I'm stumped...

**Comment by mrbobbytables** [comment]
yay for spelling mistakes on mobile lol
/label tide/merge-method-squash

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:80]: Consider using site-relative link here.

```suggestion
[this blog post](/blog/2018/07/20/the-history-of-kubernetes-the-community-behind-it/).
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:123]: Consider using site-relative link here.

```suggestion
* December 2020 — [Dockershim is deprecated](/blog/2020/12/18/kubernetes-1.20-pod-impersonation-short-lived-volumes-in-csi/)  in 1.20
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:159]: Consider using site-relative link here.

```suggestion
[largest migration in Kubernetes history](/blog/2024/05/20/completing-cloud-provider-migration/),
```

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:173]: Consider using site-relative link here.

```suggestion
out the [blog post](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) and
```

**Comment by mrbobbytables** [comment]
/approve
/lgtm
/hold
I'll leave it to @sftim to release the hold or merge other suggestions beforehand and re-lgtm

File: content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md
Comment (mengjiao-liu) [inline content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:60]: Why is this paragraph the same as L74-83 below?

Comment (mengjiao-liu) [reply content/en/blog/_posts/2024-06-06-10-Years-of-Kubernetes/index.md:60]: PR https://github.com/kubernetes/website/pull/46696 has resolved it.

**Comment by ricardoamaro** [comment]
Seems like despite having made a small contribution in https://docs.google.com/document/d/12Z43mBReJTesjbw2SbbGNPTiqYdwoBxFPC9iLRELGdc I came late to the PR party.  Thank you all for merging it!

**Comment by ricardoamaro** [comment]
Filed a PR for a key milestone: March 2018 Kubernetes becomes the first graduated CNCF project https://github.com/kubernetes/website/pull/46744

**Pull request merged**
